### DISCUSSION TITLE
Unable to succesfully read any info from the Alfen Wallbox.

### DISCUSSION THREAD

**Discussion by CorneelBracke** [leeyuentuen/alfen_wallbox#140] [Q&A]
Hi,

I installed the alfen_wallbox using the HACS and was able to succesfully create an integration.
However no info gets propagated, and all values are unknown.
![image](https://github.com/user-attachments/assets/c8c802c4-6e0e-404d-87df-e82d6b87c935)

I have deleted the sensor and tried again a few times, but get the same outcome every time.
I have not messed with the settings yet, but i don't think this is necessary or will even work.

Has anyone had the same issue?
All help is welcome!

Kind regards,
Corneel

**Comment by leeyuentuen** [comment]
what do you see in the log

**Reply by CorneelBracke** [reply]
Nothing at all sadly

**Reply by leeyuentuen** [reply]
you are using the latest public version? to be sure it is not the transaction loop 2.8.9?

**Comment by mihmihmih1988** [comment]
1.You need to setup your integration with user "admin" and the default password you received from the installer. Do not use username you use for the apps.
2. You can not login with home assistant if you are already logged in with the app. First in, first served.

**Reply by babavers** [reply]
What if there is no password (old firmware 4.5.17 that I don't dare to update) ? User and password are required fields and if I fill out anything, it doesn't work. Any suggestions?

**Reply by johnvanderster** [reply]
I have the same problem (version 4.7.1), it has a backoffice, I don't want to update/change anything. Is there a solution?

**Reply by babavers** [reply]
> I have the same problem (version 4.7.1), it has a backoffice, I don't want to update/change anything. Is there a solution?

Oh yes. Just update and don't worry. The Alfen update note is a bit unclear. I understood that there was some kind of hidden password or a password automatically generated from the serial number that only Alfen could know. So I asked Alfen to provide me that password before doing the update. The answers I got were worthless, even after clearly explaining what I feared and after asking if my fear was correct. I gave up asking them questions allthough they answer promptly. But somehow I found on another website, I don't remember which one, more information. Long story short : just update and you'll find out during the update that the software will ask you to define the machine password by yourself. There is nothing hidden or generated by the update itself. But it's important not to forget the password you invent. Write it down on a safe place or enclose it into the wallbox or something. Otherwise even an Alfen professional will not be able anymore to change anything anymore. 
Afterwards you'll have a machine completely up to date and you will be able to make the HA-connection.
And don't worry about the backoffice configuration. That won't be disturbed or erased by the update.
Important remark : be aware that the HA Device connection uses the same interface as the ACE Service Installer or MyEve app and since only one can be connected at a time don't forget to shut down the ACE Service Installer and/or the MyEve app before doing the configuration in HA. And vice versa.
